ProjFS.Mac/PrjFSKextTests/KauthHandlerTests.mm

```diff
@@ -642,4 +642,13 @@ - (void)testShouldHandleFileOpEvent_VnodeCacheUpdated {
     }
 }
 
+- (void)testResizePendingRenamesRace
+{
+    // No assertions needed here, the assertions in the kext code and mock environment suffice
+    InitPendingRenames();
+    ResizePendingRenames(16);
+    ResizePendingRenames(16);
```

Comment:
Would it be possible to test that s_pendingRenames is indeed 16, as is the actual array size?

Reply:
Yes, can do. Really, we should be doing this all in a generic resizeable array data structure, so we can test that in isolation and don't have to go through all the contortions of exposing the guts of our specific data structure to the test framework.